service/src/test/java/bio/terra/pipelines/service/PipelinesServiceTest.java

```diff
@@ -97,7 +111,7 @@ void addPipelineInput() {
     PipelineInputDefinition newInput = new PipelineInputDefinition();
     newInput.setPipelineId(pipeline.getId());
     newInput.setName("newInput");
-    newInput.setType("Int");
+    newInput.setType("INTEGER");
```

Comment:
Should these be coming from `PipelineInputTypesEnum`?

Reply:
yes they totally should! nice catch, thanks